### ISSUE TITLE
[Bug]: Video playback issue on Artix Linux. Only youtube videos load normally.

### ISSUE THREAD

**Issue by cekimilf** [Floorp-Projects/Floorp#1264]
### Pre-Submission Checklist

- [X] I have checked the existing/closed issues to ensure this issue has not already been reported.
- [X] I am using the latest supported version of the browser.
- [X] I have tried disabling all browser extensions/add-ons to rule out interference from these.
- [X] I have tried clearing the browser cache and cookies.
- [X] I have checked with a new profile and the issue still occurs.
- [X] I have provided detailed step-by-step instructions on how to reproduce the issue.
- [X] I have included relevant screenshots or console outputs.
- [X] I have checked this problem is not occurring in the latest version of Firefox ESR.
- [X] This issue is specific to this browser and does not occur in other browsers.

### Bug Summary

Video playback doesn't work on any website other than Youtube, with the only other exception being the streaming website Twitch.tv where only video plays, but without any audio. 

This bug seems to happen only on my Artix Linux installation.

### Reproduction Steps

1. Boot up an Arch based Linux distribution (In my case, its Artix)
2. Attempt to play any video that is not on the www.youtube.com website (Such as Odysee, Bitchute, Rumble, etc...)

### Expected Behavior

Videos should play without any issue regardless of the website.

### Actual Behavior

Videos fail to load, showing a black screen. Sometimes, it gives the user the following error: "The media playback was aborted due to a corruption problem or because the media used features your browser did not support." 

On Twitch, its sometimes load the video, but without any audio. Even youtube occasionally runs into some troubles playing the video and audio at the same time. 

Are these playback issues cause by specific unsupported  types of video formats or codecs?

![image](https://github.com/Floorp-Projects/Floorp/assets/77472306/a306f5f0-7e7f-41f5-8c79-6c00d1b40117)

![image](https://github.com/Floorp-Projects/Floorp/assets/77472306/baad496e-6cee-49e4-b18b-18141d3f43df)

![image](https://github.com/Floorp-Projects/Floorp/assets/77472306/417feab4-80b9-46a8-914e-611065df7e3f)

### Screenshots

Screenshot of an error that I got in the console when trying to play a random video on Odysee.

![image](https://github.com/Floorp-Projects/Floorp/assets/77472306/474136a3-a89f-4395-be06-363fdb5ff624)

### Browser Version

11.14.1 (64-bit)

### Operating System

Artix Linux

### Additional Context

I have checked all the settings, ran different profiles, reinstalled, checked if the lack of Widevine DRM plugin was perhaps causing the issue...but nothing seems to work. I also don't have any kind of plugins installed.

This seems to be an isolated problem present only on the Arch version of the Floorp browser I downloaded from the AUR. This version of the browser works fine on my Windows 10 installation.

Additionally, the newest Firefox installation from the AUR works fine on my Artix OS, so this must be an issue with the Arch version of this specific browser.

Some of the websites I tried are: Bitchute, Odysee, Rumble and a few other video streaming sites. 

PS: Video and audio permissions are allowed in the settings.

**Comment by surapunoyousei** [comment]
Is it wayland?

**Comment by cekimilf** [comment]
> Is it wayland?

It's xorg.

**Comment by regularentropy** [comment]
I can suppose that it's a codec-related issue.

**Comment by lwb-2021** [comment]
> I can suppose that it's a codec-related issue.
Yes. Firefox 115 ESR is so old that it cannot decode many kinds of videos.Maybe the problem can be solved when floorp is updated to firefox 128 ESR

**Comment by Waffuls** [comment]
I have the same problem, and its a codec issue where floorp is missing aac support
Firefox
![Screenshot 2024-07-08 at 21-56-16 Troubleshooting Information](https://github.com/Floorp-Projects/Floorp/assets/137366565/fc0280e2-c356-43df-b7d6-6aa70cd444e2)
Floorp
![screenshot_2024-07-08-223611](https://github.com/Floorp-Projects/Floorp/assets/137366565/dd3c697c-c4e8-47b7-9c7c-93aae910a020)

**Comment by cekimilf** [comment]
I believe you guys might be right. Thank you for sharing your experience.

It must be something to do with the codec support and the Firefox version. This should probably get updated as soon as possible.

**Comment by tharrmeehan** [comment]
Can confirm this one as well.

**Comment by Axenide** [comment]
Here on Arch too.

**Comment by kaos-55** [comment]
The problem is caused by an update of ffmpeg to version 7, firefox esr (on which floorp is based) also has the same problem, I had already reported this problem some time ago as well https://github.com/Floorp-Projects/Floorp/issues/1241 
Since Arch and its forks update packages faster than many distributions and that is why in other distributions this problem may have gone unnoticed.
The solution is to downgrade the ffmpeg package to version 6 and wait until Floorp version 12 which will be based on the current version of Firefox and which does not have this problem.

**Comment by UltraBlackLinux** [comment]
same here, and I don't think I want to wait another month for floorp v12

**Comment by menndouyukkuri** [comment]
@UltraBlackLinux The developers are not just there to support you. Don't be rude to them just because you don't like them.

**Comment by jonzn4suse** [comment]
Yeah all the sites work for me.
![floorp_bitchute_Screenshot_20240915_194049](https://github.com/user-attachments/assets/af38ebe5-15f6-4c5c-95b8-65359cac0363)
![floorp_rumble_Screenshot_20240915_193924](https://github.com/user-attachments/assets/a417d3a3-91cb-47e2-bb18-cc1a1c658c3c)
![floorp_rumble_Screenshot_20240915_193848](https://github.com/user-attachments/assets/59b800c4-b59d-494c-a839-55958a721080)


Operating System: openSUSE Tumbleweed 20240912
KDE Plasma Version: 6.1.5
KDE Frameworks Version: 6.5.0
Qt Version: 6.7.2
Kernel Version: 6.10.9-1-default (64-bit)
Graphics Platform: X11
Processors: 16 × AMD Ryzen 7 5825U with Radeon Graphics
Memory: 62.1 GiB of RAM
Graphics Processor: AMD Radeon Graphics
Manufacturer: HP
Product Name: HP ProBook 455 15.6 inch G9 Notebook PC
